src/app/query/Config.query.js

```diff
@@ -17,6 +17,24 @@ export class ConfigQuery {
             .addFieldList(this._getStoreListFields());
     }
 
+    getCheckoutAgreements() {
+        return new Field('checkoutAgreements')
```

Comment:
I have not seen this field implementation enywhere, is it default?

Reply:
Yes, it is provided by default